bucket/newer-tanooki.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "1.0",
+    "description": "A NewerSMBW Save Editor.",
+    "homepage": "https://github.com/RedStoneMatt/NewerTanooki",
+    "url": "https://github.com/RedStoneMatt/NewerTanooki/releases/download/v1.0/NewerTanooki.rar",
+    "hash": "cbab72a7644bae3bf9f5ef901a85709f0f2e91b58fc6dea6f0a010073164877b",
+    "license": "Visable Source",
```

```suggestion
    "license": "GPL-3.0-only",
```
NewerTanooki's license has been set